swaap/swaap/Storyboards/Profile.storyboard

```diff
@@ -1,8 +1,9 @@
 <?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I'm not sure how much I actually edited this storyboard (and some of these changes might be from the profileView branch)